pkgs/applications/video/obs-studio/plugins/obs-replay-source.nix

```diff
@@ -18,6 +18,9 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-MzugH6r/jY5Kg7GIR8/o1BN36FenBzMnqrPUceJmbPs=";
   };
 
+  # obs development headers depend on uthash but they are not in the output
+  env.NIX_CFLAGS_COMPILE = "-I${obs-studio.src}/deps/uthash";
```

Comment:
This won't be necessary anymore with obs-replay-source 1.6.13:

https://github.com/exeldro/obs-replay-source/commit/584c028bca8161efb32d72a0d31ba3071cfad706

Reply:
https://github.com/NixOS/nixpkgs/pull/277396

Reply:
Merged in https://github.com/NixOS/nixpkgs/pull/277670

Reply:
Thanks for pointing this out; I just rebased and dropped the commit that made the changes that are no longer necessary and it indeed builds on my machine.